sdks/python/setup.py

```diff
@@ -35,11 +35,32 @@
     cythonize = lambda *args, **kwargs: []
 
 
-def get_version():
-  global_names = {}
-  execfile(os.path.normpath('./apache_beam/version.py'),
-           global_names)
-  return global_names['__version__']
+# Reads the actual version from pom.xml file,
+def get_version_from_pom():
+  with open('pom.xml', 'r') as f:
+    pom = f.read()
+    regex = '.*<parent>\s*<groupId>[a-z\.]+</groupId>\s*<artifactId>[a-z\-]+</artifactId>\s*<version>([0-9a-zA-Z\.\-]+)</version>.*'
+    pattern = re.compile(regex)
+    search = pattern.search(pom)
+    return search.group(1).replace("-SNAPSHOT", ".dev")  # TODO: PEP 440 and incubating suffix
```

Comment:
hopefully if we're copying version from Maven pom then we get the incubating suffix for free?